auslib/db.py

```diff
@@ -1040,11 +1060,33 @@ def addRelease(self, name, product, blob, changed_by, transaction=None):
     def updateRelease(self, name, changed_by, old_data_version, product=None, read_only=None, blob=None, transaction=None):
         if product or blob:
             self._proceedIfNotReadOnly(name, transaction=transaction)
+        where = [self.name == name]
         what = {}
-        if read_only is not None:
-            what['read_only'] = read_only
+
+        current_product = self.select(where=where, columns=[self.product], transaction=transaction)[0]["product"]
```

Comment:
Ah, I see. You're saying that we can't restrict access to _only_ the read_only flag as things stand now. Since we don't really know what our automation use cases here will be, I'll add a comment about maybe wanting to change this later but leave the code the same.